Dshop/apps/products_catalogue/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+
+class CategoryBaseModel(models.Model):
+    name = models.CharField(max_length=200, unique=True)
+    slug = models.SlugField(max_length=200, unique=True)
+    is_active = models.BooleanField(default=True)
+    sort = models.IntegerField(default=99)
+
+    class Meta:
+        ordering = ('sort',)
+        abstract = True
+
+    def __str__(self):
+        return self.name
+
```

Comment:
Please add slug creation on save. It will help in importing products:
Something like:

```suggestion
    def save(self, *args, **kwargs): 
        if not self.slug:
            self.slug = slugify(self.name)
        return super().save(*args, **kwargs)
```

Reply:
Added, thx for suggestion